src/Aspire.Dashboard/Components/ResourcesGridColumns/ResourceNameDisplay.razor

```diff
@@ -1,10 +1,24 @@
-﻿@using Aspire.Dashboard.Model
+﻿@namespace Aspire.Dashboard.Components
+
+@using Aspire.Dashboard.Model
+@using Aspire.Dashboard.Otlp.Model
 @using Aspire.Dashboard.Resources
 @inject IStringLocalizer<Columns> Loc
 
-<FluentStack Orientation="Orientation.Horizontal" VerticalAlignment="VerticalAlignment.Center">
-    <span><FluentHighlighter HighlightedText="@FilterText" Text="@FormatName(Resource)" /></span>
-</FluentStack>
+@{
+    var unviewedErrorCount = GetUnviewedErrorCount(Resource);
+}
+
+@if (unviewedErrorCount > 0)
+{
+    <FluentCounterBadge Style="padding: 3px 3px;" Max="9" BackgroundColor="@Color.Error" Color="Color.Fill" Appearance="Appearance.Lightweight" Count="@unviewedErrorCount" HorizontalPosition="100">
+        <a href="@GetResourceErrorStructuredLogsUrl(Resource)"><FluentHighlighter HighlightedText="@FilterText" Text="@FormatName(Resource)"/></a>
```

Comment:
Include error count in tooltip. Double check it compiles. Could move building the string to a method.
```suggestion
        <a href="@GetResourceErrorStructuredLogsUrl(Resource)" title="@(FormatName(Resource) + Environment.Newline + $"{unviewedErrorCount} errors")"><FluentHighlighter HighlightedText="@FilterText" Text="@FormatName(Resource)"/></a>
```